module/VuFind/src/VuFind/Controller/Feature/ConfigPathTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * VuFind Action Feature Trait - Configuration file path methods
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Controller_Plugins
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFind\Controller\Feature;
+
+use VuFindSearch\Command\AlphabeticBrowseCommand;
+
+/**
+ * VuFind Action Feature Trait - Configuration file path methods
+ *
+ * @category VuFind
+ * @package  Controller_Plugins
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+trait ConfigPathTrait
+{
+    /**
+     * Get path to base configuration file
+     *
+     * @param string $filename Configuration file name
+     *
+     * @return string
+     */
+    protected function getBaseConfigFilePath(string $filename): string
+    {
+        $resolver = $this->serviceLocator->get(\VuFind\Config\PathResolver::class);
+        return $resolver->getBaseConfigPath($filename);
+    }
+
+    /**
+     * Get path to local configuration file
+     *
+     * @param string $filename Configuration file name
+     *
+     * @return string
+     */
+    protected function getLocalConfigFilePath(string $filename): string
```

Comment:
is there a reason not to proxy the other parameters of the method?

Reply:
Yes, these were meant to be simple wrappers to repeated calls with same parameters. I added a clarifying comment.

Reply:
This still feels potentially confusing to me, since we're proxying a method using the same name but changing the default behavior in just one context. Could we tweak the method name for clarity? Maybe `getForcedLocalConfigFilePath`? Or even `getLocalConfigFullPath` which is not more specific but is at least distinct.